### ISSUE TITLE
fping shows packet loss, but tcpdump shows normal

### ISSUE THREAD

**Issue by yinyaliang** [schweikert/fping#270]
#### fping command
```
 /usr/sbin/fping -q -b 2000 -c 5 -i 1000 172.16.250.12
```

#### tcpdump command
```
tcpdump -i eth1 icmp and host 172.16.250.12
```

#### symptoms
 *fping command show the results below,but when I use tcpdump capture icmp packet, the cap file show normaly*
```
/usr/sbin/fping -q -b 2000 -c 5 -i 1000 172.16.250.12
172.16.250.12 : xmt/rcv/%loss = 5/4/20%, min/avg/max = 2.59/2.60/2.61
```

#### fping version
```
# fping -v
fping: Version 4.1
fping: comments to david@schweikert.ch
```
#### package %loss
```
![image](https://user-images.githubusercontent.com/5956373/209062307-4b717246-44b3-4b5e-97a7-778d3010bedd.png)
```

![image](https://user-images.githubusercontent.com/5956373/209061409-afd4bbd1-825e-4c54-b2fe-6725e1562ede.png)

**Comment by ziqingkeshi** [comment]
@yinyaliang hello yinyaliang, I also meet the problem that fping shows packet loss, but the ping shows normal,even I use the fping again shows normal.Are you resolved this problem?Thank you.
